x/ownership/client/rest/query.go

```diff
@@ -0,0 +1,39 @@
+package rest
```

Comment:
just a reminder: this implementation is not done.

Reply:
What do you mean? It is done. Here only parameters are exported, but the `list/` route is already exported as well. Open & write what missng in your opinion.

Reply:
where is the route query exported?
don't confuse handler querier (backend) and rest query (client)